diff --git a/node_modules/expo-modules-jsi/apple/scripts/build-xcframework.sh b/node_modules/expo-modules-jsi/apple/scripts/build-xcframework.sh
index 56bcb4c..d520aca 100755
--- a/node_modules/expo-modules-jsi/apple/scripts/build-xcframework.sh
+++ b/node_modules/expo-modules-jsi/apple/scripts/build-xcframework.sh
@@ -229,6 +229,9 @@ build_slice() {
     DEBUG_INFORMATION_FORMAT=dwarf-with-dsym \
     COMPILER_INDEX_STORE_ENABLE=NO \
     SWIFT_COMPILATION_MODE=wholemodule \
+    CODE_SIGNING_ALLOWED=NO \
+    CODE_SIGNING_REQUIRED=NO \
+    CODE_SIGN_IDENTITY="" \
   )
 
   local product_path="${BUILD_PRODUCTS_PATH}/${build_dir_name}"
